src/test/suite/extension.test.ts

```diff
@@ -253,7 +253,7 @@ suite(`Integration Tests`, () => {
     test(`404 status code`, async () => {
         try {
             await fileDownloader.downloadFile(
-                Uri.parse(`http://httpstat.us/404`),
+                Uri.parse(`https://github.com/microsoft/cascadia-code/releases/download/v2005.15/doesntexist`),
```

Comment:
@ross-p-smith FYI just leaving here as a note, github would rate limit if download without auths using Github actions. I think since not many people working on this repo should be okay. May be in the future we can introduce auths.

Reply:
Added #46